WebApplication/Job/ExportDrawingJobItem.cs

```diff
@@ -42,12 +42,11 @@ public override async Task ProcessJobAsync(IResultSender resultSender)
 
             Logger.LogInformation($"ProcessJob (ExportDrawing) {Id} for project {ProjectId} started.");
 
-            //await ProjectWork.ExportDrawingAsync(ProjectId, _hash);
-            Thread.Sleep(1000);
+            var url = await ProjectWork.ExportDrawingViewablesAsync(ProjectId, _hash);
```

Comment:
This is not yet working. Showing viewables in the viewer will be done in the next subtask.